storybook/cypress/e2e/viewer-grid.cy.ts

```diff
@@ -59,24 +59,21 @@ describe("Viewer Grid", () => {
     cy.get("dso-viewer-grid").shadow().find(".overlay-close-button").should("be.focused");
   });
 
-  it.skip("should emit closeOverlay on escape", () => {
-    cy.visit(urlOverlayOpened);
-    cy.get("dso-viewer-grid").then((c) => {
-      c.get(0).addEventListener("dsoCloseOverlay", cy.stub().as("closeOverlay"));
-    });
-    cy.wait(100);
-    cy.realPress("Tab"); // We're not focused in the page on visit.
-    cy.wait(100);
-    cy.realPress("Escape");
-    cy.get("@closeOverlay").should("have.been.calledOnce");
-  });
-
-  it("should trap focus on overlay open", () => {
-    cy.visit(urlOverlayOpened);
-    cy.get("dso-viewer-grid").shadow().find(".overlay-close-button").should("be.focused").realPress("Tab");
-    cy.get("dso-viewer-grid").find("div[slot = 'overlay'] a[href]").eq(0).should("be.focused").realPress("Tab");
-    cy.get("dso-viewer-grid").find("div[slot = 'overlay'] a[href]").eq(1).should("be.focused").realPress("Tab");
-    cy.get("dso-viewer-grid").shadow().find(".overlay-close-button").should("be.focused");
+  it("should emit closeOverlay on escape", () => {
+    cy.visit(url)
+      .get("dso-viewer-grid")
+      .then((c) => {
+        c.get(0).addEventListener("dsoCloseOverlay", cy.stub().as("closeOverlay"));
+      })
+      .invoke("attr", "overlay-open", "")
+      .get("dso-viewer-grid")
+      .shadow()
+      .find(".overlay")
+      .should("exist")
+      .and("have.attr", "open")
+      .realPress("Escape")
```

Comment:
Kunnen we nog iets met het testen van de focus voordat we op Escape drukken? Het is beter om dit soort aannames te testen om flaky tests te voorkomen.